js/nav-category.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function () {
+  const navCategory = $('.nav-category');
+  if (navCategory.length > 0) {
+    const linkSearch = navCategory.find('.link-search');
+    if (linkSearch) {
+      navCategory.find('.link-search').on('click', function () {
+        const searchBox = navCategory.find('.search-box');
+        if (searchBox.length) {
```

Comment:
we don't need this check since you already checked for .link-search and the box come along with it